### PR TITLE
elastic_faq

### PR DESCRIPTION
**Pull request by SethSanchez**
Works with our current doc builds, ready to merge.

### CODE REVIEW

File: source/elastic_faq.rst [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+
+- Select the default Elasticsearch zone, but we'll be launching in other zones soon!
+
+- Select the best Memory and Storage size for your application.
+
+.. image:: images/create_elastic.png
+   :align: center
+   :height: 631px
+   :width: 500 px
+   :scale: 70%
```
Comment (TheDodd) [inline source/elastic_faq.rst]: The sphinx theme we are currently using is mobile friendly. We should test this on a cell phone or two to make sure we are not breaking the responsive layout of the theme with hard coded `px` values. We can also use percentages if we need to.

File: source/elastic_faq.rst [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+
+.. image:: images/add_user_elastic.png
+   :align: center
+
+
+3. Add an ACL
+~~~~~~~~~~~~~
+
+- Under the heading **Security**, you have the option to **Add ACL**. This is necessary as we don't allow any access by default so you need to add any appropriate ACL's for your servers connecting to ObjectRocket. There are two fields: **IP Address** and **Description**. Only IP is mandatory, but a description can certainly help if you plan to have more than a few.
+
```
Comment (TheDodd) [inline source/elastic_faq.rst]: @SethSanchez might want to mention something about the ACL `transport` option as well.

**Comment by TheDodd** [comment]
+1 after the inline note about ACL `transport` option.

**Comment by SethSanchez** [comment]
Done!

**Comment by TheDodd** [comment]
+1 let's do this.

**Pull request merged**